src/DocumentFormat.OpenXml/XmlPath.cs

```diff
@@ -21,9 +21,13 @@ namespace DocumentFormat.OpenXml
     public class XmlPath
     {
         /// <summary>
-        /// Initializes a new instance of the XmlPath.
+        /// Initializes a new instance of the XmlPath from the specified
```

Comment:
can you use `<see href=""/>` annotations for types here?

Reply:
You want me to use `<see href=""/>` here or `<see cref=""/>`?  If the former, what url should I be using for the classes?

Reply:
haha yeah cref

Reply:
Cool, just making sure.  I'll work on that.

Reply:
Should be updated as requested now.